addons/account/tests/test_tax.py

```diff
@@ -65,14 +67,82 @@ def _check_compute_all_results(self, taxes, expected_values, price_unit, **kwarg
             self.assertEqual(round(values['base'], 6), tax_expected_values[0])
             self.assertEqual(round(values['amount'], 6), tax_expected_values[1])
 
-    def _check_tax_results(self, taxes, expected_values, price_unit, evaluation_context_kwargs=None, compute_kwargs=None):
-        """ Evaluate the taxes computation and compare the results with the expected ones.
+    def _prepare_taxes_computation_test(self, taxes, expected_values, price_unit, evaluation_context_kwargs=None, compute_kwargs=None):
+        evaluation_context_kwargs = evaluation_context_kwargs or {}
+        quantity = evaluation_context_kwargs.pop('quantity', 1)
+        compute_kwargs = compute_kwargs or {}
+        is_round_globally = evaluation_context_kwargs.get('rounding_method') == 'round_globally'
+        return {
+            'expected_values': expected_values,
+            'params': {
+                'test': 'taxes_computation',
+                'tax_values_list': taxes._convert_to_dict_for_taxes_computation(),
+                'price_unit': price_unit,
+                'quantity': quantity,
+                'evaluation_context_kwargs': evaluation_context_kwargs,
+                'compute_kwargs': compute_kwargs,
+                'is_round_globally': is_round_globally,
+            },
+        }
+
+    def _prepare_adapt_price_unit_to_another_taxes_test(self, price_unit, original_taxes, new_taxes, expected_price_unit):
+        return {
+            'expected_price_unit': expected_price_unit,
+            'params': {
+                'test': 'adapt_price_unit_to_another_taxes',
+                'original_tax_values_list': original_taxes._convert_to_dict_for_taxes_computation(),
+                'new_tax_values_list': new_taxes._convert_to_dict_for_taxes_computation(),
+                'price_unit': price_unit,
+            },
+        }
+
+    def _add_test_py_results(self, test):
```

Comment:
`PLZ KEEP BOTH METHODS CONSISTENT WITH EACH OTHERS.`? :smile: (I don't think that "please" is that much longer than "plz" :wink: